vowpalwabbit/explore/include/vw/explore/explore_internal.h

```diff
@@ -14,8 +16,13 @@
 #include <stdexcept>
 #include <vector>
 
-namespace exploration
+namespace VW
+{
+namespace explore
+{
+namespace details
 {
+
```

Comment:
are these randomness generators only used during exploration?

Reply:
yes looks like it

Reply:
Yeah, I want to converge on the same impl rather than a copy paste as is here. But I'll do that in another PR.